pcweb/pcweb.py

```diff
@@ -76,17 +76,17 @@
     ("/docs/database", "/docs/database/overview"),
     ("/docs/hosting", "/docs/hosting/self-hosting"),
     ("/docs/advanced-guide", "/docs/advanced-guide/custom-vars"),
-    ("/docs/library/typography", "/docs/library/typography/text"),
-    ("/docs/library/forms", "/docs/library/forms/form"),
-    ("/docs/library/layout", "/docs/library/layout/box"),
-    ("/docs/library/navigation", "/docs/library/navigation/link"),
-    ("/docs/library/datadisplay", "/docs/library/datadisplay/badge"),
-    ("/docs/library/graphing", "/docs/library/graphing/chart"),
-    ("/docs/library/disclosure", "/docs/library/disclosure/accordion"),
-    ("/docs/library/feedback", "/docs/library/feedback/alert"),
-    ("/docs/library/media", "/docs/library/media/image"),
-    ("/docs/library/overlay", "/docs/library/overlay/alertdialog"),
-    ("/docs/library/other", "/docs/library/other/html"),
+    # ("/docs/library/typography", "/docs/library/typography/text"),
```

Comment:
Why do we remove these?

Reply:
Now we have the component landing page sections for those links